src/main/java/net/minestom/server/inventory/Inventory.java

```diff
@@ -156,48 +153,18 @@ public boolean addViewer(@NotNull Player player) {
     @Override
     public boolean removeViewer(@NotNull Player player) {
         final boolean result = this.viewers.remove(player);
-        setCursorItem(player, ItemStack.AIR);
         this.clickProcessor.clearCache(player);
         return result;
     }
 
-    /**
-     * Gets the cursor item of a viewer.
-     *
-     * @param player the player to get the cursor item from
-     * @return the player cursor item, air item if the player is not a viewer
-     */
-    public @NotNull ItemStack getCursorItem(@NotNull Player player) {
```

Comment:
Both of these should come back, be deprecated, and proxy to the player inventory